stan/math/prim/scal/fun/grad_reg_lower_inc_gamma.hpp

```diff
@@ -0,0 +1,163 @@
+#ifndef STAN_MATH_PRIM_SCAL_FUN_LOWER_REG_INC_GAMMA_HPP
+#define STAN_MATH_PRIM_SCAL_FUN_LOWER_REG_INC_GAMMA_HPP
+
+#include <boost/math/tools/promotion.hpp>
+#include <stan/math/prim/scal/fun/lgamma.hpp>
+#include <stan/math/prim/scal/fun/gamma_p.hpp>
+#include <stan/math/prim/scal/fun/inv.hpp>
+#include <stan/math/prim/scal/fun/log1p.hpp>
+#include <stan/math/prim/scal/fun/digamma.hpp>
+#include <stan/math/prim/scal/fun/is_nan.hpp>
+#include <stan/math/prim/scal/fun/grad_reg_inc_gamma.hpp>
+#include <stan/math/prim/scal/err/domain_error.hpp>
+#include <limits>
+
+namespace stan {
+  namespace math {
+
+    /**
+     * The lower incomlete gamma function
```

Comment:
Please add a summary sentence: https://github.com/stan-dev/stan/wiki/How-to-Write-Doxygen-Doc-Comments